CHANGELOG.md

```diff
@@ -15,6 +15,32 @@ Changes are grouped as follows
 
 ## Next release
 
+### Changed
+
+ * Allow any interval to be configured as a string in addition to integers, so e.g. upload intervals can be configured
+   as
+   ```yaml
+   upload-interval: 2m
+   ```
+   with `s`/`m`/`h`/`d` being valid units, and `s` being implied when units are missing (to preserve backwards 
+   compatibility with old config files). If your extractor reads from these config fields directly you need to update
+   to read the `seconds` (or the computed `minutes`, `hours` or `days` ) attribute instead of the fields directly, for
+   example:
```

Comment:
This is strictly speaking breaking. Although I expect most users will not read values directly like this, I think we might want to bump major versions on this one.